x/ibc/02-client/client/utils/utils.go

```diff
@@ -103,7 +103,7 @@ func QueryTendermintHeader(cliCtx context.CLIContext) (tendermint.Header, int64,
 	}
 
 	height := info.Response.LastBlockHeight
-	prevheight := height - 1
+	prevheight := height - 1 // TODO: should this be next height?
```

Comment:
Interested to hear what this is supposed to be

Reply:
I think we it's just `height` now that we added ICS07

Reply:
cc: @AdityaSripal